scrapy/core/scheduler.py

```diff
@@ -36,10 +36,12 @@ class Scheduler:
     Also, it handles dupefilters.
     """
     def __init__(self, dupefilter, jobdir=None, dqclass=None, mqclass=None,
-                 logunser=False, stats=None, pqclass=None, crawler=None):
+                 logunser=False, stats=None, pqclass=None, dpqclass=None,
+                 crawler=None):
```

Comment:
In its current form, this change is backward incompatible, because scheduler classes are allowed to receive parameters by position. It should be:

```suggestion
                 logunser=False, stats=None, pqclass=None, crawler=None,
                 dpqclass=None):
```